src/integration-test/java/uk/ac/ebi/tsc/tesk/AuthIgnoreGroupMembershipIT.java

```diff
@@ -0,0 +1,533 @@
+
+        mockKubernetes.givenThat(
+                WireMock.get("/apis/batch/v1/namespaces/default/jobs?labelSelector=job-type%3Dtaskmaster" +
+                        "%2Ccreator-user-id%3D123")
+                        .willReturn(aResponse().withBodyFile("list/taskmasters.json")));
+        MockUtil.mockListTaskKubernetesResponses(this.mockKubernetes);
+
+        performListTask(4);
+
+    }
+
+    @Test
+    public void groupAdmin_getList() throws Exception {
+
+        mockElixir.givenThat(
+                WireMock.get("/")
+                        .willReturn(okJson("{\"sub\" : \"123\",  \"eduperson_entitlement\" : [\"urn:geant:elixir-europe.org:group:elixir:GA4GH:GA4GH-CAP:EBI:TEST:ADMIN#perun.elixir-czech.cz\"]}")));
+
+        mockKubernetes.givenThat(
+                WireMock.get("/apis/batch/v1/namespaces/default/jobs?labelSelector=job-type%3Dtaskmaster" +
+                        "%2Ccreator-group-name%20in%20%28TEST%29%2Ccreator-user-id%3D123")
+                        .willReturn(aResponse().withBodyFile("list/taskmasters.json")));
+        MockUtil.mockListTaskKubernetesResponses(this.mockKubernetes);
+
+        performListTask(4);
+
+    }
+
+    @Test
+    public void mixed_getList() throws Exception {
+
+        mockElixir.givenThat(
+                WireMock.get("/")
+                        .willReturn(okJson("{\"sub\" : \"123\",  \"eduperson_entitlement\" : [\"urn:geant:elixir-europe.org:group:elixir:GA4GH:GA4GH-CAP:EBI:ABC:ADMIN#perun.elixir-czech.cz\",\"urn:geant:elixir-europe.org:group:elixir:GA4GH:GA4GH-CAP:EBI:TEST#perun.elixir-czech.cz\"]}")));
+
+        mockKubernetes.givenThat(
+                WireMock.get("/apis/batch/v1/namespaces/default/jobs?labelSelector=job-type%3Dtaskmaster" +
+                        "%2Ccreator-group-name%20in%20%28TEST%2CABC%29%2Ccreator-user-id%3D123")
+                        .willReturn(aResponse().withBodyFile("list/taskmasters.json")));
+        MockUtil.mockListTaskKubernetesResponses(this.mockKubernetes);
+
+        performListTask(3);
+
+    }
+
+    @Test
+    public void unauthenticated_getList() throws Exception {
+
+        this.mvc.perform(get("/v1/tasks")
+                .header("Authorization", "different BAR"))
+                .andExpect(status().isUnauthorized());
+        this.mvc.perform(get("/v1/tasks?view=BASIC")
+                .header("Different", "Bearer BAR"))
+                .andExpect(status().isUnauthorized());
+        this.mvc.perform(get("/v1/tasks?view=FULL"))
+                .andExpect(status().isUnauthorized());
+
+    }
+
+    @Test
+    public void nonmember_getList() throws Exception {
+
+        mockElixir.givenThat(
+                WireMock.get("/")
+                        .willReturn(okJson("{\"sub\" : \"123\",  \"eduperson_entitlement\" : [\"sth\",\"urn:geant:elixir-europe.org:group:elixir:GA4GH:GA4GH-CAP#perun.elixir-czech.cz\"]}")));
+
```

Comment:
Doesn't the non-member belong here to the same group as member (in fact being a member)?